src/pybind/mgr/volumes/module.py

```diff
@@ -223,30 +223,43 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'perm': 'r'
         },
         {
-            'cmd': 'nfs export create '
-            'name=type,type=CephString '
+            'cmd': 'nfs export create cephfs '
```

Comment:
Wait...why are we removing the "type=" argument here? I thought we agreed to keep it? If you're removing that, then what purpose does the "cephfs" word serve here?

Reply:
> Wait...why are we removing the "type=" argument here? I thought we agreed to keep it? 

It would be good to have separate interface for cephfs and rgw. As rgw will require different set of options.

> If you're removing that, then what purpose does the "cephfs" word serve here?

The interface would be similar to fs but with different options.
`nfs export create rgw <options> ...`

Reply:
Ok, so you're basically turning the "type" option from a named parameter to a required positional one. Fair enough.